docs/source/examples/notebooks/getting_started/tutorial-4-setting-parameter-values.ipynb

```diff
@@ -377,121 +372,28 @@
-      "text/plain": [
-       "<Figure size 576x288 with 2 Axes>"
-      ]
-     },
-     "metadata": {
-      "needs_background": "light"
-     },
-     "output_type": "display_data"
-    }
-   ],
-   "source": [
-    "model = pybamm.lithium_ion.SPMe()\n",
-    "sim = pybamm.Simulation(model, parameter_values=parameter_values)\n",
-    "sim.solve()\n",
-    "sim.plot([\"Current [A]\", \"Voltage [V]\"])"
+    "and we observe in the plot that now the applied current is indeed 10 A. Note that we still passed the interval `[0, 3600]` to `sim.solve()`, but the simulation terminated early as the lower voltage cut-off was reached."
    ]
   },
   {
    "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Custom current function"
+    "### Function parameters"
    ]
   },
   {
    "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Alternatively, we can define the current to be an arbitrary function of time"
+    "Some parameters, like the current, sometimes need to be defined as a function (in this case a function of time). As an example, we now want to set the current to be sinusoidal. To do this, we need to define the relevant function and pass it as a parameter."
```

Comment:
Can we reword this sentence: "Some parameters, like the current, sometimes need to be defined as a function (in this case a function of time)."

Between the some and sometimes, plus the parentheses, it seems like this got a bit unwieldy

Reply:
Maybe: "Some parameters can be defined as a function. In this example we will make current a function of time."

Reply:
When dealing with Rob's comments I ended up rephrasing the paragraph so this is gone now.